app/models/tasks/copy_media_task.rb

```diff
@@ -82,10 +87,9 @@ def porter_copy_task
   end
 
   def porter_slice_task
-    input_opts = []
-    input_opts << "-ss #{media_resource.slice_start}" if media_resource.slice_start.present?
-    input_opts << "-to #{media_resource.slice_end}" if media_resource.slice_end.present?
     output_opts = ["-map_metadata 0"]
+    output_opts << "-ss #{media_resource.slice_start}" if media_resource.slice_start.present?
+    output_opts << "-to #{media_resource.slice_end}" if media_resource.slice_end.present?
```

Comment:
And this is #940 - still works, and I don't see a noticeable difference in resulting files.  So ... success?